docs/cron-workflows.md

```diff
@@ -109,6 +110,41 @@ For example, with `timezone: America/Los_Angeles`:
     |            | 2        | 2020-11-02 02:01:00 -0800 PST |
     |            | 3        | 2020-11-03 02:01:00 -0800 PST |
 
+#### Skip forward (missing schedule)
```

Comment:
Per https://github.com/argoproj/argo-workflows/pull/13365#discussion_r1790563038, even if you left these split, I would still simplify these, per the style guide, and still use the word "leap" as that is the common term for this and not "skip" (which has other connotations)